### ISSUE TITLE
Create a bootstrap that allows creation of contract-specific command line tools

### ISSUE THREAD

**Issue by frol** [near/devx#207]
I believe that it will greatly improve the DevX of contracts if developers can have a way to easily bootstrap a CLI tool for a contract (to have a dedicated tool instead of using near-shell with manual JSON crafting).

Thus, instead of

```
near call my_validator deposit '{}' --accountId user1 --amount 100
near call my_validator stake '{"amount": "100000000000000000000000000"}' --accountId user1
```

a contract user will have a specific interface, thus minimizing the scope (no need to understand and remember the JSON format) and reducing the room for an error:

```
npx near-staking-pool my_validator deposit-and-stake 100N
```

/cc @evgenykuzyakov @potatodepaulo @nearmax @mikedotexe @amgando

**Comment by willemneal** [comment]
This issue is somewhat related and more general purpose for calling any contract: https://github.com/near/near-shell/issues/367

**Comment by frol** [comment]
@willemneal that API might still be too low-level. Yet, I agree that we can explore the space of generic solutions as well, but in order to make is user-friendly we need to solve the [contract metadata problem (a.k.a. contract interfaces)](https://github.com/nearprotocol/nearcore/issues/1972)

**Comment by vgrichina** [comment]
@frol do you mean instantiating dependency objects automatically a-la `near repl` or something else?

**Comment by frol** [comment]
@vgrichina Yes, `near repl`-like instantiation with CLI framework in place, so developers will only need to add new CLI subcommands calling `await contract.welcome({ account_id: "frol" })`
